src/stepper.js

```diff
@@ -1,6 +1,12 @@
+/* @flow */
 const errorMargin = 0.0001;
 
-export default function stepper(frameRate, x, v, destX, k, b) {
+export default function stepper(frameRate: number,
+                                x: number,
+                                v: number,
+                                destX: number,
+                                k: number,
+                                b: number): Array<number> {
```

Comment:
It does not seem to check for index (I tested with this: https://gist.github.com/ludovicofischer/a53742cc8d884512ba7b)